setup/db/create-schema.sql

```diff
@@ -2280,6 +2280,8 @@ CREATE TABLE `cloud`.`s2s_customer_gateway` (
   `domain_id` bigint unsigned NOT NULL,
   `account_id` bigint unsigned NOT NULL,
   `removed` datetime COMMENT 'date removed if not null',
+  `ike_version` varchar(5) NOT NULL DEFAULT 'ike' COMMENT 'one of ike, ikev1, ikev2',
+  `split_connections` int(1) NOT NULL DEFAULT 0,
```

Comment:
@ggoodrich-ipp 
this affects only fresh installation.

need to add sqls in engine/schema/src/main/resources/META-INF/db/schema-41510to41600.sql
```
alter table s2s_customer_gateway add column `ike_version` varchar(5) NOT NULL DEFAULT 'ike' COMMENT 'one of ike, ikev1, ikev2';
alter table s2s_customer_gateway add column `split_connections` int(1) NOT NULL DEFAULT 0;
```

Reply:
Thanks @weizhouapache, we hadn't caught that. Those changes are now included.